components/info/ErrorMessage.tsx

```diff
@@ -0,0 +1,10 @@
+import clsx from 'clsx'
+
+import Panel from '@/components/shared/Panel'
+import Typography from '@/components/shared/Typography'
+
+export const ErrorMessage = ({ message, className }: { className?: string; message: string }) => (
```

Comment:
Please follow consistent code style (https://bratislava.github.io/frontend/components):
- default export instead of named export
- props type as separate type (usually called `ErrorMessagePanelProps`, or just `Props`, if not exported)
- more explicit component name, such as `ErrorMessagePanel`